src/transformers/__init__.py

```diff
@@ -453,6 +453,7 @@
         "is_vision_available",
         "logging",
     ],
+    "utils.bitsandbytes_utils": [],
```

Comment:
```suggestion
    "utils.bitsandbytes": [],
```
No need to repeat `utils`.

Reply:
fixed in 39c46a07648fa5565117c6ac33bfcbe69449f2c0